### ISSUE TITLE
expect_identical_xl() calls tinytest::expect_equivalent()

### ISSUE THREAD

**Issue by jdblischak** [eddelbuettel/ttdo#6]
I was surprised that `expect_identical_xl()` calls `tinytest::expect_equivalent()`. Is this intended behavior?

https://github.com/eddelbuettel/ttdo/blob/bf8db554b61394d3adb0b78cbbf470cba485f50b/R/expectations_xl.R#L52-L57

Here is a reproducible example that demonstrates that `ttdo::expect_identical_xl()` and `tinytest::expect_identical()` can give different results:

```
x <- rnorm(10)
y <- x + 1e-10

library(tinytest)
using(ttdo)

expect_equal(x, y)
expect_equal_xl(x, y)
expect_equal_with_diff(x, y)

expect_identical(x, y)
expect_identical_xl(x, y)
```

Running on my machine, I get the following:

```
> x <- rnorm(10)

> y <- x + 1e-10

> library(tinytest)

> using(ttdo)

> expect_equal(x, y)
----- PASSED      : <-->
 call| expect_equal(x, y) 

> expect_equal_xl(x, y)
----- PASSED      : <-->
 call| expect_equal_xl(x, y) 

> expect_equal_with_diff(x, y)
----- PASSED      : <-->
 call| expect_equal_with_diff(x, y) 

> expect_identical(x, y)
----- FAILED[data]: <-->
 call| expect_identical(x, y)
 diff| TRUE 

> expect_identical_xl(x, y)
----- PASSED      : <-->
 call| expect_identical_xl(x, y)
```

Here's my session information:

```
> sessionInfo()
R version 3.6.3 (2020-02-29)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 18.04.4 LTS

Matrix products: default
BLAS:   /usr/lib/x86_64-linux-gnu/blas/libblas.so.3.7.1
LAPACK: /usr/lib/x86_64-linux-gnu/lapack/liblapack.so.3.7.1

locale:
 [1] LC_CTYPE=C.UTF-8       LC_NUMERIC=C           LC_TIME=C.UTF-8        LC_COLLATE=C.UTF-8    
 [5] LC_MONETARY=C.UTF-8    LC_MESSAGES=C.UTF-8    LC_PAPER=C.UTF-8       LC_NAME=C             
 [9] LC_ADDRESS=C           LC_TELEPHONE=C         LC_MEASUREMENT=C.UTF-8 LC_IDENTIFICATION=C   

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] ttdo_0.0.5     tinytest_1.2.2

loaded via a namespace (and not attached):
[1] compiler_3.6.3  parallel_3.6.3  tools_3.6.3     rstudioapi_0.11 crayon_1.3.4   
[6] diffobj_0.2.4  
```

**Comment by eddelbuettel** [comment]
@barbehenna Do you remember?  I (very vaguely) recall us having a discussion on corner cases, but this looks like a copy-and-paste-oh.  Thoughts?

**Comment by barbehenna** [comment]
I suspect that is likely a copy-paste error. I think most of our focus was on expect_equivalent_xl versus expect_equal_xl, so it's possible this one slipped through the cracks.

**Comment by eddelbuettel** [comment]
That's what it looks like.  I just sent a hot fix up which tests fine so I think I'll sent it out as 0.0.6.  Unless there are objections...

**Comment by eddelbuettel** [comment]
I shipped 0.0.6 to CRAN, and given the [clean sheet](https://cloud.r-project.org/web/checks/check_results_ttdo.html) and lack of depends it should just sail through but seemingly the processing stopped -- as it sometimes happens.  Likely going to be up tomorrow.

Thanks again to @jdblischak for spotting this!

**Comment by jdblischak** [comment]
@eddelbuettel Thanks for the quick fix!
